### PR TITLE
GitHub: Added ps1 scripts to simplify CI/CD

### PR DESCRIPTION
**Pull request by 0x5bfa**
## Summary

As the title.

## PR Checklist

- [x] Were these changes approved in an issue or discussion with the project maintainers? In order to prevent extra work, feature requests and changes to the codebase must be approved before the pull request will be reviewed. This prevents extra work for the contributors and maintainers.

### CODE REVIEW

File: scripts/Submit-ToMicrosoftStore.ps1
Comment (yaira2) [inline scripts/Submit-ToMicrosoftStore.ps1:1]: I think we should link to the original source in order to credit the author.

File: scripts/Configure-AppxManifest.ps1
Comment (yaira2) [inline scripts/Configure-AppxManifest.ps1:1]: This looks nice!

File: scripts/Submit-ToMicrosoftStore.ps1
Comment (0x5bfa) [reply scripts/Submit-ToMicrosoftStore.ps1:1]: Added.

File: .github/workflows/deploy-preview.yml
Comment (yaira2) [inline .github/workflows/deploy-preview.yml:1]: Can you fork these changes into a seperate PR?

File: .github/workflows/deploy-stable.yml
Comment (yaira2) [inline .github/workflows/deploy-stable.yml:1]: As above

File: scripts/Configure-AppxManifest.ps1
Comment (yaira2) [inline scripts/Configure-AppxManifest.ps1:1]: As above

**Comment by 0x5bfa** [comment]
I mean I could but it will take a week because I don’t have time much these days. I’m pretty satisfied changes in this PR and this won’t affect on the app codebase.

**Comment by 0x5bfa** [comment]
If I do have write permission I would not have hurdle when I test those for you because if so I can create a branch in this repo and run workflow manually. While developing this function can I have? It reduces many steps you and I have now that are troublesome.

**Comment by yaira2** [comment]
> I mean I could but it will take a week because I don’t have time much these days. I’m pretty satisfied changes in this PR and this won’t affect on the app codebase.

I'm confident as well and I'm ready to merge those changes, but the Store action is definitely not going to work as is so we might as well split the changes into two PRs.

**Comment by 0x5bfa** [comment]
That's true. Thanks, I will do that then.

**Comment by 0x5bfa** [comment]
Done. Deleted store-related changes and left only scripts.

**Review by yaira2 — APPROVED** [review]
LGTM

**Pull request merged**